test/compflags/lydia/library/SKIPIF

```diff
@@ -0,0 +1,2 @@
+# Does not currently work with multilocale
```

Comment:
GASNet complains "message layer not initialized" when we try to call through it without having started launching through it.  @gbtitus was saying he thought it would require additional effort to get this to work.  I didn't actually try ugni . . .